src/components/Profile/Profile.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import { rockets } from "../../features/rockets/rocketSlice";
+import "./Profile.css";
+
+const Profile = () => {
+  const data = useSelector(rockets);
+  const missions = useSelector((state) => state.missions.missions);
+
+  return (
+    <div className="profile__cont">
+      <div>
+        <h2>My Missions</h2>
+        <div className="profiles">
+          {missions
+            .filter((x) => x.reserved === true)
+            .map((b) => (
+              <div className="profile" key={b.mission_name}>
+                {b.mission_name}
+              </div>
+            ))}
+        </div>
+      </div>
```

Comment:
- [ ] As stated in the project requirement in the section requirement  Please ensure that in the 'Render UI: My Profile' section, the list of all joined missions is displayed correctly. Additionally, it is crucial that when I navigate to another section (e.g., 'Rocket' or 'Mission'), the status of joined missions should not reset to 'Not Member.' Furthermore, please ensure that the data is not retained in the profile section of the 'Missions' column layout. For a visual reference, please watch the [video](https://drive.google.com/file/d/1MOTo_76zL03qPLZSNM_WgnmSqJbpyKOt/view?usp=sharing) provided below, which demonstrates the issues that need to be fixed